jaeger-core/src/test/java/com/uber/jaeger/reporters/RemoteReporterTest.java

```diff
@@ -68,10 +72,13 @@ public void setUp() throws Exception {
   public void testRemoteReporterReport() throws Exception {
     Span span = (Span) tracer.buildSpan("raza").start();
     reporter.report(span);
-    Thread.sleep(50);
+    // do sleep until automatic flush happens on 'reporter'
+    // added 20ms on top of 'flushInterval' to avoid corner cases
+    await().with().pollInterval(50, TimeUnit.MILLISECONDS).atMost(
+            flushInterval + 20, TimeUnit.MILLISECONDS).until(() -> sender.getReceived().size() > 0);
```

Comment:
@yurishkuro as `flushInterval`is 1000ms. flush might happen on any ms in the window of 1000ms, for corner case I have added another 20ms. Here I can reduce `pollInterval`.